src/mixins/filing-template-mixin.ts

```diff
@@ -988,16 +988,9 @@ export default class FilingTemplateMixin extends DateMixin {
   formatEmptyRestoration (filing: any): RestorationFilingIF {
     const toReturn = filing
     if (toReturn.restoration) {
-      // set offices
-      if (!toReturn.restoration?.offices) {
-        toReturn.restoration.offices = []
-      }
       // set contact point
       if (!toReturn.restoration?.contactPoint) {
-        toReturn.restoration.contactPoint = {
-          email: '',
-          phone: ''
-        }
+        toReturn.restoration.contactPoint = EmptyContactPoint
```

Comment:
This is an object assignment. I believe that if you did
```
toReturn.restoration.contactPoint.email = 'severin@example.com`
```
that it would update the Empty Contact Point object itself.

Instead, you can spread the empty object into a new object (or use cloneDeep if the object is more than 1 layer deep).

Reply:
Do you need to init Contact Point here, or is its initialization in the store OK?

Reply:
Yikes! I am thinking this again. Let me try not keeping it here and test again. It is already in state-model. 
![image](https://user-images.githubusercontent.com/116035339/216670583-b3d3fb94-1119-4e08-a5d3-2427e251bab8.png)

Reply:
It's called "iterating", and the code keeps getting better 😆

Reply:
initialization in the store works well with a null/empty check for first load. Do not need formatEmptyRestoration call.

Reply:
Excellent - one less special case!